tox.ini

```diff
@@ -10,12 +10,16 @@ setenv =
   LANGUAGE=en_US
   LC_ALL=en_US.utf-8
   SETUPTOOLS_ENABLE_FEATURES=legacy-editable
+allowlist_externals = mv
 deps =
   -r requirements-dev.txt
 commands =
     python setup.py bdist_wheel -- -- -j4
     pip install --find-links={toxinidir}/dist qiskit_aer
+    mv qiskit_aer qiskit_aer.bak
     stestr run {posargs}
+commands_post =
+    mv qiskit_aer.bak qiskit_aer
```

Comment:
Thanks. I don't think my solution is nice, it's workaround, and I don't want to merge this PR. Please make new one.

I open this PR until it will be fixed (for users who want to know this workaround.).